developer-docs-site/static/examples/python/first_nft.py

```diff
@@ -1,358 +0,0 @@
-            "0x3::token::CollectionData",
-            collection_name.encode("utf-8").hex(),
-        )
-
-    # <:!:section_3
-
-    def cancel_token_offer(
-        self,
-        account: Account,
-        receiver: str,
-        creator: str,
-        collection_name: str,
-        token_name: str,
-    ):
-        payload = {
-            "type": "entry_function_payload",
-            "function": f"0x3::token_transfers::cancel_offer_script",
-            "type_arguments": [],
-            "arguments": [
-                receiver,
-                creator,
-                collection_name.encode("utf-8").hex(),
-                token_name.encode("utf-8").hex(),
-                0,
-            ],
-        }
-        self.submit_transaction_helper(account, payload)
-
-
-if __name__ == "__main__":
-    client = TokenClient(TESTNET_URL)
-    faucet_client = FaucetClient(FAUCET_URL, client)
-
-    alice = Account()
-    bob = Account()
-    jack = Account()
-    collection_name = "Alice's cat collection"
-    token_name = "Alice's tabby"
-
-    print("\n=== Addresses ===")
-    print(f"Alice: {alice.address()}")
-    print(f"Bob: {bob.address()}")
-    print(f"Jack: {bob.address()}")
-
-    faucet_client.fund_account(alice.address(), 10_000_000)
-    faucet_client.fund_account(bob.address(), 10_000_000)
-    faucet_client.fund_account(jack.address(), 10_000_000)
-
-    print("\n=== Initial Balances ===")
-    print(f"Alice: {client.account_balance(alice.address())}")
-    print(f"Bob: {client.account_balance(bob.address())}")
-    print(f"Jack: {client.account_balance(jack.address())}")
-
-    print("\n=== Creating Collection and Token ===")
-
-    client.create_collection(
-        alice, collection_name, "Alice's cat collection", "https://aptos.dev"
-    )
-    client.create_token(
-        alice,
-        collection_name,
-        token_name,
-        "Alice's simple token",
-        1,
-        "https://aptos.dev/img/nyan.jpeg",
-    )
-    print(
-        f"Alice's collection: {client.get_collection(alice.address(), collection_name)}"
-    )
-    print(
-        f"Alice's token balance: {client.get_token_balance(alice.address(), alice.address(), collection_name, token_name)}"
-    )
-    print(
-        f"Alice's token data: {client.get_token_data(alice.address(), collection_name, token_name)}"
-    )
-
-    print("\n=== Transferring the token to Bob ===")
-    client.offer_token(
-        alice, bob.address(), alice.address(), collection_name, token_name, 1
-    )
-    client.claim_token(
-        bob, alice.address(), alice.address(), collection_name, token_name
-    )
-    print(
-        f"Alice's token balance: {client.get_token_balance(alice.address(), alice.address(), collection_name, token_name)}"
-    )
-    print(
-        f"Bob's token balance: {client.get_token_balance(bob.address(), alice.address(), collection_name, token_name)}"
-    )
-
-    print("\n=== On-chain lazy mint through semi-fungible token ===")
```

Comment:
Please keep the example

Reply:
Let's resurrect it in the python sdk. This isn't the right place for it.